apps/davi/src/utils/guildsProposals.ts

```diff
@@ -10,3 +10,12 @@ export function mapStructToProposal(
   };
   return proposal;
 }
+

```

Comment:
should we call this file something like "daosProposals"?

Reply:
I moved it into the more general `utils/proposals` file, and renamed it `getGuildOptionLabel`, since it's tightly related to how we label guild proposals (we assume that the 0 option is "against").